Dockerfile

```diff
@@ -1,46 +1,40 @@
-# VERSION 0.1
 # AUTHOR:           Clay Teeter <teeterc@gmail.com>, Nicholas Long <nicholas.long@nrel.gov>
 # DESCRIPTION:      Image with seed platform and dependencies running in development mode
 # TO_BUILD_AND_RUN: docker-compose build && docker-compose up
 
-# This Dockerfile has been updated to pull from our last known good build of SEED (v2.6.1).
-# Version 3.7.2-r2 of geos has introduced and incompatible library:
-#    https://pkgs.alpinelinux.org/package/edge/testing/x86_64/geos
-#FROM alpine:3.8
+FROM alpine:3.14
 
-# Start with 2.6.0. note that the source code will be removed and re-copied to this container. The
-# version of SEED here is used to load in the core system packages and dependencies.
-FROM seedplatform/seed:2.6.0
-
-# DO NOT UPGRADE until libgeos and shapely fix the connection.
-#RUN apk add --no-cache python \
-#        python3-dev \
-#        postgresql-dev \
-#        alpine-sdk \
-#        pcre \
-#        pcre-dev \
-#        libxslt-dev \
-#        linux-headers \
-#        libffi-dev \
-#        bash \
-#        bash-completion \
-#        npm \
-#        nginx && \
-#    apk add --no-cache --repository http://dl-cdn.alpinelinux.org/alpine/edge/main openssl && \
-#    apk add --no-cache --repository http://dl-3.alpinelinux.org/alpine/edge/testing/ geos gdal && \
```

Comment:
we can now install these directly without a repo

Reply:
nice! this will make maintaining geos and gdal easier!